1650990475718	Sync.LogManager	DEBUG	Flushing file log
1650990475734	Sync.LogManager	DEBUG	Log cleanup threshold time: 1650126475734
1650990475746	Sync.LogManager	DEBUG	Done deleting files.
1650990532535	Sync.Service	DEBUG	User-Agent: Firefox/99.0.1 (Linux x86_64) FxSync/1.101.0.20220413030647.desktop
1650990532535	Sync.Service	INFO	Starting sync at 2022-04-26 21:58:52 in browser session O_aMM1OIq9pR
1650990532535	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1650990532537	Sync.Status	INFO	Resetting Status.
1650990532537	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1650990532537	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1650990532537	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1650990532538	Sync.ErrorHandler	ERROR	Sync encountered an error: Error: Can't sync: Network is offline(resource://services-sync/stages/enginesync.js:59:13) JS Stack trace: sync@enginesync.js:59:13
onNotify@service.js:1351:28
WrappedNotify@util.js:210:32
WrappedLock@util.js:166:27
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
1650990532539	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1650990532539	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 5554296 ms.
1650990532539	Sync.SyncScheduler	DEBUG	Next sync in 5554296 ms. (why=client-backoff-schedule)
1650990532539	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1650990532543	Sync.Service	DEBUG	Exception calling anonymous function: Error: Can't sync: Network is offline(resource://services-sync/stages/enginesync.js:59:13) JS Stack trace: sync@enginesync.js:59:13
onNotify@service.js:1351:28
WrappedNotify@util.js:210:32
WrappedLock@util.js:166:27
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
